### PR TITLE
feat: list members of a community

### PR DESCRIPTION
**Pull request by octref**

### CODE REVIEW

**Review by isTravis — APPROVED** [review]
Wonderful - really! Whether you've adapted to the codebase quickly, or this is just how you write code, this looks nearly identical to how I think about creating components, naming, layout, etc. 

I saw one or two indentation mismatches that I'm sure prettier would catch, but other than that - looks great to me!

One convention we keep (which you may have done in previous roles) is that the person who creates the PR is responsible for merging it once Approved, and unless explicit commits are needed, we do a squash+merge.

**Comment by octref** [comment]
Thanks for your positive feedback!

> I saw one or two indentation mismatches that I'm sure prettier would catch

Not sure what those are – I ran prettier through all files to double-check and there's no diff.
I added a pre-commit hook to format all files before they are commited to git, and I ran prettier through the codebase.

File: components/UserPreview/UserPreview.tsx
Comment (isTravis) [inline components/UserPreview/UserPreview.tsx:17]: Maybe it's a github rendering issue? This `<Avatar>` line was one of the indentation oddities I saw.

Comment (isTravis) [reply components/UserPreview/UserPreview.tsx:17]: Oh weird - no, it shows up locally for me in my editor, but prettier doesn't seem to fix it?

Comment (octref) [reply components/UserPreview/UserPreview.tsx:17]: @isTravis Duh – I ran prettier through ts/js/json/scss, but forgot to include `tsx`. I'll fix that in a sec.
Please see the new PR: https://github.com/underlay/r1/pull/26

**Pull request merged**
